set_vcf_sample_id/README.md

```diff
@@ -0,0 +1,185 @@
+# Update a VCF header line sample ID
+
+This pipeline is useful when you have a set of single-sample VCFs
+in Cloud Storage in which the sample ID in the header needs to be changed.
+
+For example, suppose you have a set of 24 VCFs for a single individual
+where each VCF contains variants for a single chromosome (1-22, X, Y). 
+Suppose further that the header line of each VCF looks like:
+
+```
+#CHROM POS ID REF ALT QUAL FILTER INFO FORMAT SAMPLE-001
+```
+
+and you want it to look like:
+
+```
+#CHROM POS ID REF ALT QUAL FILTER INFO FORMAT SAMPLE-001-TEST-01
+```
+
+The scripts in this example can be used to make this change to a set of VCFs.
+The input VCFs can be compressed with gzip or bzip2 or they can be uncompressed.
+The output VCFs' compression state will reflect the input VCFs'.
+
+| API Note |
+|----------|
+| This example demonstrates using the Pipelines API to run custom code *without building a Docker image*. In this example, the scripts are copied into Cloud Storage and then downloaded at pipeline run time.  The docker image used is the stock [Python 2.7 image from dockerhub](https://hub.docker.com/_/python/). |
+
+| API Note |
+|----------|
+| This example demonstrates the use of non-file input parameters. These parameters (`ORIGINAL_SAMPLE_ID` and `NEW_SAMPLE_ID`) get set in the environment and are then available to the code running in the docker container. |
+
+## (1) Copy scripts to Cloud Storage
+
+```
+gsutil cp set_vcf_sample_id.sh set_vcf_sample_id.py \
```

Comment:
I'm wrong about the typo.

But consider renaming one of these two files so that they appear to be two different files?  For example the shell script name could mention that it also takes care of compression.